transaction-service/src/main/java/pl/adamboguszewski/transactionservice/application/Transaction.java

```diff
@@ -0,0 +1,58 @@
+package pl.adamboguszewski.transactionservice.application;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Transaction {
+
+    private final List<Product> products;
+
+    private final double totalPrice;
+
+    private final String transactionId;
+
+    private final String checkoutId;
+
+    private final LocalTime transactionTime;
+
+    private final LocalDate transactionDate;
+
```

Comment:
We need more detailed informations about transaction, create another class TransactionInfo which will contain info like cashId, our transactionDateTime and some payment informations (TransactionPayment will be another class, remember that you can pay in parts so transactionPayments will be list of TransactionPayment)